src/Table/Table.tsx

```diff
@@ -0,0 +1,985 @@
+    primaryActionButton?: PrimaryActionButton;
+    selectedRow: TableRow;
+    tableRef: React.RefObject<HTMLTableElement>;
+}
+
+const ActionColumn: React.FunctionComponent<ActionColumnProps> = (props: ActionColumnProps) => {
+    const btnPrimaryRandomIds = randomId("btn");
+    const [dropup, setDropup] = React.useState<boolean>(false);
+    const actionRef: React.RefObject<HTMLDivElement> = React.createRef<HTMLDivElement>();
+
+    function getActionColumnClasses(): string {
+        let className: string = "dropdown-content";
+        if (props.selectedRow?.actionsDropdownDropped) {
+            className += " active";
+        }
+
+        if (dropup) {
+            className += " dropup";
+        }
+
+        return className;
+    }
+
+    return (
+        <div className="action-column">
+            {props.primaryActionButton &&
+                <button
+                    id={btnPrimaryRandomIds}
+                    type="button"
+                    className="btn btn-outline-primary btn-sm"
+                    onClick={(e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+                        props.primaryActionButton.onClick(e, props.selectedRow);
+                    }}
+                >
+                    {props.primaryActionButton.label}
+                </button>
+            }
+            {props.actionLinks && props.actionLinks.length > 0 &&
+                <div
+                    className="ellipsis-dropdown-holder"
+                    onClick={(e) => {
+                        const tableSize: DOMRect = props.tableRef?.current?.getBoundingClientRect();
+                        const actionColumnHeight: number = actionRef.current?.scrollHeight;
+                        const actionColumSize: DOMRect = actionRef.current?.getBoundingClientRect();
+
+                        if (tableSize?.height > actionColumnHeight) {
+                            const lengthOffset: number = tableSize?.bottom - actionColumSize?.bottom;
+                            if (lengthOffset < actionColumnHeight) {
+                                setDropup(true);
+                            } else {
+                                setDropup(false);
+                            }
+                        } else {
+                            setDropup(false);
+                        }
+                        e.preventDefault();
+                        props.onActionDropped(e);
+                    }}
+                >
+                    <div className="icon-holder" id={"ellipsis-" + props.selectedRow.rowIndex}>
+                        {ellipsis}
+                    </div>
+                    <div
+                        className={getActionColumnClasses()}
+                        ref={actionRef}
+                    >
+                        {props.actionLinks.map((link: ActionLinkItem, index: number) =>
+                            <a
+                                href="#"
```

Comment:
you should not have a `#` as a URL since it might cause a reload or unwanted scroll jumps you can just remove it and add the styles necessary